integration_tests/backend/test_backend.py

```diff
@@ -0,0 +1,73 @@
+import pytest
+import requests
+import subprocess
+from pathlib import Path
+import os
+import sys
+
+import aqueduct
+
+
+def get_response(endpoint, additional_headers={}):
+    headers = {"api-key": pytest.api_key}
+    headers.update(additional_headers)
+    url = f"{pytest.server_address}{endpoint}"
+    if url[:4] != "http":
+        try:
+           r = requests.get("https://"+url, headers=headers) 
+        except:
```

Comment:
Can we figure out what prefix to use in the  `setup_class()` method. That way we don't have to send bad requests for every `get_response()` call. You sould just pass the url to `get_response()`.